client/js/shout.js

```diff
@@ -7,7 +7,7 @@ $(function() {
 		"/devoice",
 		"/disconnect",
 		"/invite",
-		"/join",
+		"/join","/j",
```

Comment:
It shouldn't be added to the command list since it is used to autocomplete and typing `/j` and hitting tab should just autocomplete to `/join`